docs-client/package.json

```diff
@@ -26,25 +26,25 @@
     "regenerator-runtime": "^0.13.8"
   },
   "devDependencies": {
-    "@babel/core": "^7.13.14",
-    "@babel/preset-env": "^7.13.12",
+    "@babel/core": "^7.14.2",
+    "@babel/preset-env": "^7.14.2",
     "@babel/preset-react": "^7.13.13",
     "@types/favicons-webpack-plugin": "^1.0.0",
     "@types/html-webpack-plugin": "^3.2.5",
     "@types/json-bigint": "^1.0.0",
     "@types/jsonminify": "^0.4.1",
     "@types/material-ui": "^0.21.8",
     "@types/node": "^14.14.37",
-    "@types/react": "^17.0.3",
-    "@types/react-dom": "^17.0.3",
-    "@types/react-helmet": "^6.1.0",
+    "@types/react": "^17.0.5",
+    "@types/react-dom": "^17.0.5",
+    "@types/react-helmet": "^6.1.1",
     "@types/react-router-dom": "^5.1.7",
     "@types/react-select": "^4.0.14",
```

Comment:
We can actually remove this. I removed the actual library in #3167, but forgot to remove the types 😅

Reply:
Oh, nice catch!!